flow/layers/color_filter_layer_unittests.cc

```diff
@@ -82,10 +87,11 @@ TEST_F(ColorFilterLayerTest, SimpleFilter) {
   const SkRect child_bounds = SkRect::MakeLTRB(5.0f, 6.0f, 20.5f, 21.5f);
   const SkPath child_path = SkPath().addRect(child_bounds);
   const SkPaint child_paint = SkPaint(SkColors::kYellow);
-  auto layer_filter =
-      SkColorMatrixFilter::MakeLightingFilter(SK_ColorGREEN, SK_ColorYELLOW);
+
+  auto dl_color_filter = DlLinearToSrgbGammaColorFilter::instance;
   auto mock_layer = std::make_shared<MockLayer>(child_path, child_paint);
-  auto layer = std::make_shared<ColorFilterLayer>(layer_filter);
+  auto layer = std::make_shared<ColorFilterLayer>(
+      DlLinearToSrgbGammaColorFilter::instance);
```

Comment:
This is already in local variable dl_color_filter. You could reuse the local variable here for a shorter line.